spec/jobs/facilities/state_cemetery_reload_job_spec.rb

```diff
@@ -65,4 +65,10 @@
     expect(cemetery.phone['main']).to eq('251-625-1338')
     expect(cemetery.phone['fax']).to eq('251-626-9204')
   end
+
+  it 'should gracefully handle missing mailing addresses' do
```

Comment:
the description confuses me compared to the assertion

Reply:
Yeah, I guess that description doesn't make as much sense without the extra context around it throwing an error downstream.

Maybe `it 'should make empty addresses not nil'` is better? Too literal?